Common/ArmCPUDetect.cpp

```diff
@@ -290,7 +290,7 @@ std::string CPUInfo::Summarize()
 	if (bNEON) sum += ", NEON";
 	if (bIDIVa) sum += ", IDIVa";
 	if (bIDIVt) sum += ", IDIVt";
-	if (CPU64bit) sum += ", 64-bit";
+	if (CPU64bit) sum += ", 64-bit support";
```

Comment:
This will change reporting and make everything report as a new CPU.

I'd kinda prefer we don't change it, or else make sure to strip this change out in `postdata.Add("cpu", cpu_info.Summarize());`.

-[Unknown]